overrides/android-override.json

```diff
@@ -84,7 +84,12 @@
         "clientBrandHint": {
             "state": "enabled",
             "minSupportedVersion": 51852000,
-            "exceptions": [],
+            "exceptions": [
+                {
+                    "domain": "chatgpt.com",
```

Comment:
I think this has to be in "domains" not sure this does anything here.

See: https://app.asana.com/1/137249556945/project/1208578714566209/task/1205259295535506?focus=true

Reply:
Whoops 💩 sorry, ofc it does.